resources/views/book_tour/layout/menu.blade.php

```diff
@@ -24,17 +24,17 @@
                     <li>
                         <div class="row">
 
-                            @foreach($types as $type)
+                            @foreach ($types as $type)
                                 @if($type->parent_id == null)
                                 <div class="col-md-6 col-lg-3">
                                     <ul class="list-unstyled mb-3">
                                         <h5>
                                            {{ $type->name }}
                                         </h5>
-                                        @foreach($types as $childen)
+                                        @foreach ($types as $childen)
```

Comment:
Line 27 với 34 lặp giống nhau từ cái $types ? In kiểu cha con thế này k đúng rồi e

Reply:
dạ em sẽ tìm hiểu và sửa lại ạ